ynput/ayon-core
# After Effect: multiselection for file loading into the folder

## Changelog Description
This PR is to fix the issue when loading multiple files into the selected folder and only the first loaded assets get into it.

## Additional info
n/a

## Testing notes:
1. Build the aftereffect addon with this branch or use {YOUR_AYON_CORE_REPO}\server_addon\aftereffects\client\ as addon directory
2. Install addon
3. Launch After Effects -> Load
4. Create New folder
5. Select the folder
6. load file from the loader
7. load another file from the loader again (or select all the assets and load at the same time)
8. Should be all added into the folder.
